docs/_includes/star-rating.html

```diff
@@ -15,12 +17,80 @@
 
     {% highlight html %}
 <svg role="img" aria-label="Rating: 2.5/5" class="star-rating" focusable="false" height="12">
     <use xlink:href="#star-rating-2-5"></use>
 </svg>
     {% endhighlight %}
 
+    <h3>Method 2: Independent Stars</h3>
+    <p>This is built using fine-grained pieces (individual stars) for optimal flexibility and for allowing potential independant star treatments (i.e. animations).</p>
+    <p>
+        <ul class="demo-icon-boxes">
+            {% assign ratings = "0, 0-5, 1, 1-5, 2, 2-5, 3, 3-5, 4, 4-5, 5" | split: ", " %}
+            {% for rating in ratings %}
+            <li>
+                <span>{{ rating }}</span>
+                <div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="{{ rating }}" focusable="false">
+                    <svg class="star_rating_icon" aria-hidden="true">
+                        <use class="star_rating_icon_symbol" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                        <use class="star_rating_icon_star_half" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                    </svg>
+
+                    <svg class="star_rating_icon" aria-hidden="true">
+                        <use class="star_rating_icon_symbol" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                        <use class="star_rating_icon_star_half" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                    </svg>
+
+                    <svg class="star_rating_icon" aria-hidden="true">
+                        <use class="star_rating_icon_symbol" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                        <use class="star_rating_icon_star_half" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                    </svg>
+
+                    <svg class="star_rating_icon" aria-hidden="true">
+                        <use class="star_rating_icon_symbol" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                        <use class="star_rating_icon_star_half" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                    </svg>
+
+                    <svg class="star_rating_icon" aria-hidden="true">
+                        <use class="star_rating_icon_symbol" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                        <use class="star_rating_icon_star_half" xlink:href="{{ page.static_dir }}/common/icons.svg#icon-star-full"></use>
+                    </svg>
+                </div>
+            </li>
+            {% endfor %}
+        </ul>
+    </p>
+
+    {% highlight html %}
+<div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="2-5" focusable="false">
```

Comment:
`focusable` needs to go on the svg tags. Although as we don't support IE... I think it's still relevant in some versions of Edge we support